### ISSUE TITLE
DTMF tone should be defined as ABNF

### ISSUE THREAD

**Issue by marcoscaceres** [sysapps/telephony#142]
The spec should define DTMF tone as ABNF, so that it can be more easily used in various algorithms. 

> 0-9; A-D; *; #.

**Comment by marcoscaceres** [comment]
Something like. 

``` ABNF
tone = DIGIT / menu / symbols
menu = %x41-44; A-D
symbol = "*" / "#"
```

**Comment by zolkis** [comment]
Isn't DTMF already standardized?
http://www.etsi.org/technologies-clusters/technologies/fixed-line-access/dtmf

**Comment by marcoscaceres** [comment]
Yeah, but it's more so we know how to deal with things like case handling, etc.... just make it easier to write tests for.
